src/components/account/details/DetailsPageMain.js

```diff
@@ -0,0 +1,31 @@
+import React, { Component } from 'react'
+
+import EditAccountDetails from './EditAccountDetails'
+import ChangePassword from './ChangePassword'
+import Subscribe from './Subscribe'
+import EditCompany from './EditCompany'
+import CancelAccount from './CancelAccount'
+
+import '../../../styles/account/details/details_page_main.css'
+
+class DetailsPageMain extends Component {
+  render() {
+    return (
+      <div>
+        <div className="page-main">
+          <div className="header">
+            <h1 className="page-main__title">Account details</h1>
```

Comment:
For the className here and line 18, why did you choose to use underscores instead of hyphens?